src/models/utils.js

```diff
@@ -2,9 +2,11 @@ import log from 'loglevel';
 import moment from 'moment';
 
 function hideLastName(name) {
-  const fullNameArray = name.split(' ');
-  const hiddenFullName = `${fullNameArray[0]} ${fullNameArray[1][0]}`;
-  return hiddenFullName;
+  const fullNameArray = name?.split(' ');
```

Comment:
This solves an application error if a plater does not have a space in their fullName.